test/MVCFramework45.FunctionalTests/FunctionalTest/DependencyTelemetryMvcTests.cs

```diff
@@ -34,8 +37,15 @@ public void OperationIdOfRequestIsPropagatedToChildDependency()
             try
             {
                 Assert.True(telemetries.Count >= 2);
-                Assert.Equal(2, telemetries.Where((t) => t.Context?.Operation?.Name != null && t.Context.Operation.Name.Contains(path)).Count());
-                Assert.Equal(telemetries[0].Context.Operation.Id, telemetries[1].Context.Operation.Id);
+                Assert.Equal(2, telemetries.Count(t => t.Context?.Operation?.Name != null && t.Context.Operation.Name.Contains(path)));
+                var dependency = (DependencyTelemetry)telemetries.Single(t =>
+                {
+                    var dt = t as DependencyTelemetry;
+                    return dt?.Context?.Operation?.Name != null && dt.Context.Operation.Name.Contains(path);
+                });
+
+                var request = (RequestTelemetry)telemetries.Single(t => t is RequestTelemetry);
+                Assert.Equal(dependency.Context.Operation.Id, request.Context.Operation.Id);
```

Comment:
Probably yes, since we expect dependency to have the same Id as request. I'll fix that